include/fmt/format.h

```diff
@@ -3198,17 +3198,24 @@ FMT_CONSTEXPR basic_string_view<Char> compile_string_to_view(
   return {s.data(), s.size()};
 }
 
-#define FMT_STRING_IMPL(s, base)                                  \
-  [] {                                                            \
-    /* Use a macro-like name to avoid shadowing warnings. */      \
-    struct FMT_COMPILE_STRING : base {                            \
-      using char_type = fmt::remove_cvref_t<decltype(s[0])>;      \
-      FMT_MAYBE_UNUSED FMT_CONSTEXPR                              \
-      operator fmt::basic_string_view<char_type>() const {        \
-        return fmt::detail::compile_string_to_view<char_type>(s); \
-      }                                                           \
-    };                                                            \
-    return FMT_COMPILE_STRING();                                  \
+#if defined(FMT_GCC_VERSION) && FMT_GCC_VERSION >= 400
```

Comment:
Bug present only in GCC 6.4 - 7.3:
#if FMT_GCC_VERSION >= 604 && FMT_GCC_VERSION < 704

Reply:
There is no need to overspecify. The attribute is harmless.

Reply:
Ok, I leave this line as it is then, right?..

Reply:
Yes

Reply:
Actually you don't need to check the version at all because we don't support gcc < 4, so this should be just `#if FMT_GCC_VERSION`.

Reply:
done